main.py

```diff
@@ -146,20 +147,24 @@ def main(arg_list: list[str] | None = None) -> None:
 
   """
 
-  args = parse_args(arg_list)
+  try:
+    args = parse_args(arg_list)
 
-  config = build_abcd_params_config(args)
+    config = build_abcd_params_config(args)
 
-  start_time = time.time()
-  print("Starting ABCD assessment... \n")
+    start_time = time.time()
+    print("Starting ABCD assessment... \n")
```

Comment:
I think print should be avoid using to export "log" in not test/dev env. Is there any special reason to use "print"?
https://google.github.io/styleguide/pyguide.html#3101-logging